python/test/test_allVerification_bench.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+# ===========================================================
+#  File Name: test_rep_bench.py
+#  Author: Xu Zhang, Columbia University
```

Comment:
@albutko Things are looking good. Probably would be good to change this line to your name, and put Georgia Tech for the files you've written.

Reply:
(if this was a file that you wrote)